lib/asciidoctor/standoc/converter.rb

```diff
@@ -89,11 +89,26 @@ def html_doc_path(file)
       end
 
       def flavor_name
-        self.class.name.split("::")&.[](-2).downcase
+        self.class.name.split("::")&.[](-2).downcase.to_sym
       end
 
       def fonts_manifest
-        File.expand_path(File.join(@libdir, "../../metanorma/", flavor_name, "fonts_manifest.yaml"))
+        flavor = flavor_name
+        registry = Metanorma::Registry.instance
+        processor = registry.find_processor(flavor)
```

Comment:
@opoudjis @ronaldtse maybe you can suggest a better way how to access related processor?

Reply:
The processor is fine. processor.short == flavour_name, but that doesn't help you find the processor. Requiring the directory name to be the same as the flavour name (modulo capitalisation) is reasonable, and we are already presupposing it elsewhere.

Reply:
Better Processor search interface needed...